standard-wallet-backend/README.md

```diff
@@ -0,0 +1,201 @@
+Being able to configure the backend host is really important to connect a wallet to a different Cardano network, so anyone can deploy a backend for the new network and connect the wallet to it.
+
+# **Motivation**
+
+As the Cardano ecosystem grows, more Cardano networks start to appear. Leading a project called Shareslake, I deployed a Cardano mainnet, finding that allowing people to connect existing wallets to this new network is really complex.
+
+To illustrate the problem let’s use Nami wallet as example. It allows to configure the transaction submit endpoint, nevertheless, it uses Blockfrost to read on-chain data such us the funds on the wallet address. So, even though you can change the submit endpoint, the wallet won’t show the address funds or transactions to the user.
+
+This CIP is created to suggest a standard on how wallets obtain the on-chain information from the backend, allowing people to easily edit the light wallet connection settings to connect it to a different Cardano network.
+What we are looking to achieve with this CIP is already possible into the Ethereum ecosystem. Let's think on a wallet called Metamask that you can easily connect to different Ethereum networks.
+
+# **Rationale**
+
+Focusing this CIP on the communication between backends indexing the ledger and the light wallets we should be able to suggest a bunch of standard endpoints that allows light wallet's users to just change the backend host and the creators of a new network to deploy a backend knowing that it will work with most of the light wallets.
+
+Cardano mainnet addresses do not include the network magic allowing to use the same address and keys between Cardano networks, so just changing of the backend host on the wallet settings is enough. Testnet addresses are different, they include the network magic, requiring to create a new address per network.
+
+There are two different points to standardise here:
+
+1. Endpoints used to read the on-chain data from the backend.
+2. Endpoints used to submit transactions to the network.
+
+The proposal must be agnostic to the wallet and backend implementations or programming languages so we will be defining REST endpoints that will be exposed by the backends and consumed by the wallets.
+
+To define those endpoints using [OpenAPI](https://swagger.io/specification/) seems ideal as it is widely used to specify RESTful APIs and there are several tools for automatically generating APIs code.
+
+About the first point, we could follow `cardano-db-sync` REST endpoints as standard for consuming on-chain data, but they were deprecated.
+
+The endpoints proposed into the specification are based on the required by Nami wallet, which is an open source widely used Cardano light wallet that at the date of writting this CIP is using Blockfrost as backend.
```

Comment:
I would discourage using Blockfrost's api. It's not open source and it doesn't handle pagination across rollbacks.

Reply:
I agree, just added a couple of endpoints as examples to illustrate the intention of the CIP. Looks like Blockfrost plans to open-source the code regarding their website but it isn't at this moment.

Reply:
I was told about this project that could be a good alternative: https://api.koios.rest/#overview